grafana/Makefile

```diff
@@ -36,7 +36,7 @@ ifeq ($(ARCH),amd64)
 	BASEIMAGE?=busybox
 	CC=gcc
 endif
-ifeq ($(ARCH),arm)
+ifeq ($(ARCH),armhfrm)
```

Comment:
Why did you make this change? When I'm trying to build grafana containers I can see:
```
docker run --rm -it -v /tmp/tmp.OnAJjYyL8A:/build -w /go/src/github.com/grafana/grafana gcr.io/google_containers/kube-cross:v1.8.3-1 /bin/bash -c "\
		curl -sSL https://github.com/grafana/grafana/archive/v4.2.0.tar.gz | tar -xz --strip-components=1 \
		&& curl -sSL https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana_4.2.0_amd64.deb > /tmp/grafana.deb \
		&& mkdir /tmp/grafanarootfs && dpkg -x /tmp/grafana.deb /tmp/grafanarootfs \
		&& CGO_ENABLED=1 GOARCH=arm CC=cc go build --ldflags=\"-w -X main.version=v4.2.0 -X main.commit=unknown-dev -X main.timestamp=0 -extldflags '-static'\" -o /tmp/grafanarootfs/usr/sbin/grafana-server ./pkg/cmd/grafana-server \
		&& CGO_ENABLED=1 GOARCH=arm CC=cc go build --ldflags=\"-w -X main.version=v4.2.0 -X main.commit=unknown-dev -X main.timestamp=0 -extldflags '-static'\" -o /tmp/grafanarootfs/usr/sbin/grafana-cli ./pkg/cmd/grafana-cli \
		&& cd /tmp/grafanarootfs && tar -cf /build/grafana.tar . \
		&& cd /build && CGO_ENABLED=0 GOARCH=arm go build -o setup_grafana setup_grafana.go"
# runtime/cgo
cc: error: unrecognized command line option '-marm'
```